css/css-fonts/variations/font-variation-settings-inherit.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <title>Testing the inheritance of the font-variation-settings property</title>
+    <link rel="help" href="https://www.w3.org/TR/css-fonts-4/#propdef-font-variation-settings" />
+    <script src="/resources/testharness.js"></script>
+    <script src="/resources/testharnessreport.js"></script>
+</head>
+<body>
+    <div id="inheritanceTestParent" style="font-variation-settings: 'xxxx' 1, 'xxxx' 2;">
+        <span id="inheritanceTestChildInherited">Abc</span>
+        <span id="inheritanceTestChildOverride" style="font-variation-settings: 'cccc' 1, 'bbbb' 2, 'aaaa' 3, 'aaaa' calc(2 + 2);">Abc</span>
+    </div>
+    <script>
+
+        // Verify computed inheritance of nested elements.
+        var elementParent = document.getElementById("inheritanceTestParent");
+        var elementChildInherited = document.getElementById("inheritanceTestChildInherited");
+        var elementChildOverride = document.getElementById("inheritanceTestChildOverride");
+
+        var parentValue = "'yyyy' 1, 'yyyy' 2";
+        elementParent.style.fontVariationSettings = parentValue;
+
+        test(() => {
+            var actualValue = window.getComputedStyle(elementParent).fontVariationSettings;
+            assert_equals(actualValue, "\"yyyy\" 2", "Duplicate axis tags should be removed, favoring the latter axis tag's value.");
```

Comment:
It might be a result of how we store the flags in Edge, maybe we just can't store this duplication internally and therefore simplify anyway. Paging @SergeyMalkin here.

Reply:
Drive-by comment: I believe the spec should soon be updated to reflect this as correct behavior per https://github.com/w3c/csswg-drafts/issues/1959.